test-framework/junit5-internal/src/main/java/io/quarkus/test/QuarkusUnitTest.java

```diff
@@ -165,6 +170,15 @@ public QuarkusUnitTest setAfterAllCustomizer(Runnable afterAllCustomizer) {
         return this;
     }
 
+    /**
+     * Provides a convenient way to either add additional dependencies to the application (if it doesn't already contain a
+     * dependency), or override a version (if the dependency already exists)
+     */
+    public QuarkusUnitTest setForcedDependencies(List<AppArtifact> forcedDependencies) {
```

Comment:
Very nice, I think this will be really useful for testing different dependency combinations.

Reply:
:+1